src/Microsoft.ML.FastTree/TreeEnsemble/QuantileRegressionTree.cs

```diff
@@ -91,6 +97,39 @@ public void SetLabelsDistribution(double[] labelsDistribution, double[] weights)
             _instanceWeights = weights;
         }
 
+        /// <summary>
+        /// Copy training examples' labels and their weights to external variables.
+        /// </summary>
+        /// <param name="leafSamples">List of label collections. The type of a collction is a double array. The i-th label collection contains training examples' labels falling into the i-th leaf.</param>
+        /// <param name="leafSampleWeights">List of labels' weight collections. The type of a collction is a double array. The i-th collection contains weights of labels falling into the i-th leaf.
```

Comment:
>collction [](start = 94, length = 9)

typo. See line 103 as well.

Reply:
Nice catch.

---
In reply to: [252034011](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252034011) [](ancestors = 252034011)